### PR TITLE
Fix: Posts sent to forums had been rejected

### PR DESCRIPTION
**Pull request by annando**
This fixes a problem when posts that had been sent to a forum hadn't been redistributed. This - for example - touches the problem for local forums that hadn't really worked for local users.

Also this enables the forums to use AP to send posts to Friendica users.

### CODE REVIEW

File: src/Protocol/DFRN.php [hunk 1]
```diff
@@ -1500,15 +1500,20 @@ private static function fetchauthor($xpath, $context, $importer, $element, $only
...
+			$condition = DBA::mergeConditions($condition, ['rel' => [Contact::SHARING, Contact::FRIEND]]);
+		}
+
 		$contact_old = DBA::selectFirst('contact', $fields, $condition);
 
 		if (DBA::isResult($contact_old)) {
 			$author["contact-id"] = $contact_old["id"];
 			$author["network"] = $contact_old["network"];
 		} else {
+			Logger::info('Blubb', ['condition' => $condition]);
```
Comment (MrPetovan) [inline src/Protocol/DFRN.php]: Please remove this debug call.

**Review by MrPetovan — CHANGES_REQUESTED** [review]

File: src/Protocol/DFRN.php (hunk 1 above)
Comment (annando) [reply src/Protocol/DFRN.php]: Örgs :-)

Comment (annando) [reply src/Protocol/DFRN.php]: "blubb" in fact is an abreviation for ... ehemm, ... let me look ... 😁

Comment (annando) [reply src/Protocol/DFRN.php]: Done

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**